pkg/shared/src/i18n/en/index.ts

```diff
@@ -137,11 +141,225 @@ const inbound_note = {
 	}
 };
 
+const customer_orders_page = {
+	title: "Customer Orders",
+	new_order: "New Order",
+	status_filters: {
+		in_progress: "In Progress",
+		completed: "Completed"
+	},
+	customer: "Customer",
+	order_id: "Order ID",
+	update_order: "Update Order",
+	customer_details: "Customer Details",
+	update: "Update"
+};
+
+const suppliers_page = {
+	new_supplier: "New Supplier",
+	suppliers: "Suppliers",
+	delete: "Delete",
+	edit: "Edit",
+	supplierId: {
+		new_order_page: {
+			total_books: "Total books",
+			total_value: "Total value",
+			books: "Books",
+			select: "Select",
+			isbn: "ISBN",
+			title: "Title",
+			authors: "Authors",
+			ordered_quantity: "Ordered quantity",
+			total: "Total",
+			selected_quantity: "Selected quantity",
+			selected_books: "Selected books",
+			place_order: "Place Order"
+		},
+		supplier_page: "Supplier page",
+		supplier_name: "Supplier name",
+		supplier_address: "Supplier address",
+		supplier_email: "Supplier email",
+		create_new_order: "Create new order",
+		assigned_publishers: "Assigned publishers",
+		publisher_name: "Publisher name",
+		remove_publisher: "Remove publisher",
+		unassigned_publishers: "Unassigned publishers",
+		add_to_supplier: "Add to supplier"
+	},
+	orders_page: {
+		supplier_orders: "Supplier Orders",
+		suppliers: "Suppliers",
+		unordered: "Unordered",
+		ordered: "Ordered",
+		reconciling: "Reconciling",
+		completed: "Completed",
+		no_unordered_books: {
+			description: "No unordered supplier orders available. Create a customer order first to generate supplier orders.",
+			button: "New Customer Order"
+		},
+		orderId: {
+			reconciled_list: {
+				view_reconciliation: "View Reconciliation",
+				reconcile: "Reconcile",
+				total_books: "Total books",
+				total_value: "Total value",
+				ordered: "Ordered",
+				print_order: "Print Order",
+				books: "Books",
+				isbn: "ISBN",
+				title: "Title",
+				authors: "Authors",
+				quantity: "Quantity",
+				total_price: "Total Price"
+			}
+		}
+	},
+	reconcile_page: {
+		reconcile_deliveries: "Reconcile Deliveries",
+		created: "Created",
+		last_updated: "Last Updated",
+		includes_supplier_orders: "Includes supplier orders",
+		no_books: "Scan or enter the ISBNs of the delivered books to begin reconciliation.",
+		isbn: "ISBN",
+		title: "Title",
+		authors: "Authors",
+		quantity: "Quantity",
+		price: "Price",
+		total_delivered: "Total delivered",
+		delete_dialog: {
+			confirmDeleteDialogHeading: "Delete Reconciliation Order",
+			confirmDeleteDialogDescription:
+				"Are you sure you want to delete this reconciliation order? This action will delete all the scanned lines."
+		}
+	}
+};
+
+const warehouses = {
+	warehouse_id: {
+		display_label: {
```

Comment:
`column_headers`, or whatever other standard label you've used for another table?